src/main/java/org/jabref/model/entry/field/UnknownField.java

```diff
@@ -47,7 +54,7 @@ public int hashCode() {
     @Override
     public String toString() {
         return "UnknownField{" +
-                "name='" + name + '\'' +
-                '}';
+               "name='" + name + '\'' +
```

Comment:
one ' is escaped, the other not, because of surrounding double quotes. technically correct, but it hurts my eyes.

Reply:
Probably auto generated by the IDE(A)